rbi/stdlib/weakref.rbi

```diff
@@ -0,0 +1,35 @@
+# typed: __STDLIB_INTERNAL
+
+# Weak Reference class that allows a referenced object to be garbage-collected.
+#
+# A [`WeakRef`](https://docs.ruby-lang.org/en/2.6.0/WeakRef.html) may be used
+# exactly like the object it references.
+#
+# Usage:
+#
+# ```ruby
+# foo = Object.new            # create a new object instance
+# p foo.to_s                  # original's class
+# foo = WeakRef.new(foo)      # reassign foo with WeakRef instance
+# p foo.to_s                  # should be same class
+# GC.start                    # start the garbage collector
+# p foo.to_s                  # should raise exception (recycled)
+# ```
+class WeakRef
```

Comment:
When I tested this on Stripe's CI, it said that `WeakRef` descends from `Delegator`, which i confirmed via IRB:

```
❯ irb
irb(main):001:0> require 'weakref'
=> true
irb(main):002:0> WeakRef.ancestors
=> [WeakRef, Delegator, #<Module:0x00007fcfee88a8e0>, BasicObject]
```

```suggestion
class WeakRef < Delegator
```